app/src/main/java/com/amaze/filemanager/ui/activities/FlashScreen.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2014-2022 Arpit Khurana <arpitkh96@gmail.com>, Vishal Nehra <vishalmeham2@gmail.com>,
+ * Emmanuel Messulam<emmanuelbendavid@gmail.com>, Raymond Lai <airwave209gt at gmail.com> and Contributors.
+ *
+ * This file is part of Amaze File Manager.
+ *
+ * Amaze File Manager is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.amaze.filemanager.ui.activities;
+
+import com.amaze.filemanager.R;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.animation.AnimationUtils;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class FlashScreen extends AppCompatActivity {
+  TextView text1;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_flash_screen);
+
+    text1 = findViewById(R.id.flash_text);
+    text1.startAnimation(AnimationUtils.loadAnimation(this, android.R.anim.slide_in_left));
+
+    Intent intent = new Intent(this, MainActivity.class);
+    Thread thread =
+        new Thread() {
+          public void run() {
+            try {
+              sleep(1000);
+            } catch (Exception exception) {
+              exception.printStackTrace();
+            } finally {
+              text1.clearAnimation();
+              startActivity(intent);
+              finish();
```

Comment:
Start activity should be done before the animation so that it loads as the animation plays.